crates/model/src/order.rs

```diff
@@ -283,10 +283,13 @@ pub struct OrderCreation {
 }
 
 #[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
 pub struct OrderCreationPayload {
     #[serde(flatten)]
     pub order_creation: OrderCreation,
     pub from: Option<H160>,
+    #[serde(default)]
+    pub is_liquidity_order: bool,
```

Comment:
Make `is_liquidity_order` optional with `false` as default to minimize breaking backwards compatibility.

Reply:
Is this todo? Not sure If I should review the PR or wait until you push something more?